php/meterpreter/ext_server_stdapi.php

```diff
@@ -429,25 +429,40 @@ function add_stat_buf($path) {
     if ($st) {
         $st_buf = "";
         $st_buf .= pack("V", $st['dev']);
-        $st_buf .= pack("v", $st['ino']);
-        $st_buf .= pack("v", $st['mode']);
-        $st_buf .= pack("v", 0);
-        $st_buf .= pack("v", $st['nlink']);
-        $st_buf .= pack("v", $st['uid']);
-        $st_buf .= pack("v", $st['gid']);
+        $st_buf .= pack("V", $st['mode']);
+        $st_buf .= pack("V", $st['nlink']);
+        $st_buf .= pack("V", $st['uid']);
+        $st_buf .= pack("V", $st['gid']);
         $st_buf .= pack("V", $st['rdev']);
-        $st_buf .= pack("V", $st['size']);
-        $st_buf .= pack("V", $st['ctime']);
-        $st_buf .= pack("V", $st['atime']);
-        $st_buf .= pack("V", $st['mtime']);
+
+        $st_buf .= pack_p($st['ino']);
+        $st_buf .= pack_p($st['size']);
+        $st_buf .= pack_p($st['atime']);
+        $st_buf .= pack_p($st['mtime']);
+        $st_buf .= pack_p($st['ctime']);
```

Comment:
Aligns with the [Java Meterpreter implementation](https://github.com/rapid7/metasploit-payloads/blob/master/java/meterpreter/stdapi/[%E2%80%A6]main/java/com/metasploit/meterpreter/stdapi/stdapi_fs_stat.java) and the [Python Meterpreter implementation](https://github.com/rapid7/metasploit-payloads/blob/0339cb0e31b0ea283401df7bed6f5dfb6fa3bd1f/python/meterpreter/ext_server_stdapi.py#L897-L899) - although we noticed the Python implementation doesn't send blocks/blocksize